x/ecocredit/core/msg_mint_batch_credits.go

```diff
@@ -92,11 +92,13 @@ func validateOriginTx(o *OriginTx, required bool) error {
 		}
 		return nil
 	}
-	if !reOriginTxNote.MatchString(o.Typ) {
-		return errBadReq.Wrap("origin_tx.typ must be 2-64 long, valid characters: alpha-numberic, space, '-' or '_'")
-	}
-	if !reOriginTxNote.MatchString(o.Id) {
+	// TODO: should we limit this to transaction format for polygon as well?
+	if !reOriginTxId.MatchString(o.Id) {
```

Comment:
@clevinson thoughts? Should we limit the transaction format for ethereum transaction ids? I'm leaning towards yes but just want to double check.

Reply:
Not here I don't think. We will at some point be supporting Verra tokenization directly.